### DISCUSSION TITLE
我阿里腾讯云cf都有在用 可以实现三个服务商同时监听修改吗

### DISCUSSION THREAD

**Discussion by kukuos** [jeessy2/ddns-go#373] [Q&A]
我阿里腾讯云cf都有在用 可以实现三个服务商同时监听修改吗

**Comment by kukuos** [comment]
![image](https://user-images.githubusercontent.com/109440295/188561327-d7c3a303-edff-4fe6-a4e0-029e761eae27.png)

**Comment by jeessy2** [comment]
不支持。可以转到一个服务商
